apps/backend/src/modules/quiz-answers/quiz-answer.service.ts

```diff
@@ -143,9 +143,9 @@ class QuizAnswerService implements Service {
 		);
 
 		await this.categoryService.deleteUserScores(userId);
-		const { scores } = await this.createScores({ answerIds, userId });
+		const { items } = await this.createScores({ answerIds, userId });
 
-		return { scores, userAnswers };
+		return { scores: items, userAnswers };
```

Comment:
What was the need to rename scores to items?

Reply:
For consistency. In all other methods, where we are returning arrays, like findAll's, we return {items : <specific type>[]}